keyboards/sha/info.json

```diff
@@ -0,0 +1,110 @@
+{
+    "manufacturer": "Steven Karrmann",
+    "keyboard_name": "Sha",
+    "maintainer": "skarrmann",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["GP8", "GP9", "GP10", "GP11", "GP12", "GP13", "GP14"],
+        "rows": ["GP28", "GP4", "GP27", "GP5", "GP26", "GP6", "GP15", "GP7"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "animations": {
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "knight": true,
+            "christmas": true,
+            "static_gradient": true,
+            "rgb_test": true,
+            "alternating": true,
+            "twinkle": true
+        },
+        "default": {
+            "val": 0
+        },
+        "layers": {
+            "enabled": true
+        },
```

Comment:
This would need to be set at the keymap level, or there needs to be code at the kb level.

Reply:
Thanks @drashna! Enabling RGB Lighting Layers has been removed from the keyboard info.json, and moved into the default keymap config.h. RGB Lighting Layers is now defined solely at the keymap level.